test/integration/api/OrderAPI.spec.ts

```diff
@@ -894,6 +894,49 @@ describe('OrderAPI', () => {
         );
       });
     });
+
+    describe('when liquidity amounts don’t match amount of components of issuance order', async () => {
+      beforeEach(async () => {
+        const takerWalletOrder = subjectOrders[0] as TakerWalletOrder;
+        takerWalletOrder.takerTokenAmount = takerWalletOrder.takerTokenAmount.sub(1);
+        subjectOrders[0] = takerWalletOrder;
+      });
+
+      test('throws', async () => {
+        const takerWalletOrder = subjectOrders[0] as TakerWalletOrder;
+
+        const {
```

Comment:
Can combine this with the above line.

const {
  takerTokenAddress,
  takerTokenAmount,
} = subjectOrders[0] as TakerWalletOrder;